src/packages.builds

```diff
@@ -5,11 +5,13 @@
   <PropertyGroup>
     <AdditionalProperties>BuildPackageLibraryReferences=false</AdditionalProperties>
     <PackageReportDir Condition="'$(PackageReportDir)' == ''">$(BinDir)pkg/reports/</PackageReportDir>
+    <BuildAllOSGroups Condition="'$(OSGroup)' != '' OR '$(FilterToOSGroup)' != ''">false</BuildAllOSGroups>
```

Comment:
OSGoup is never going to be empty as it is always defaulted to AnyOS in dir.props if it is empty so I don't think that part of the condition is necessary.